### ISSUE TITLE
Add support of system font values on font property

### ISSUE THREAD

**Issue by regseb** [stylelint/stylelint#3793]
The rule `font-family-no-missing-generic-family-keyword` returns false positives when [system font values](https://developer.mozilla.org/en-US/docs/Web/CSS/font#System_font_values) are used in `font` property.

```css
input {
  font: caption;
}
```

```json
{
  "rules": {
    "font-family-no-missing-generic-family-keyword": true
  }
}
```

> Which version of stylelint are you using?

e.g. `9.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint "**/*.css" --config myconfig.json`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

e.g. "Yes, it's related to SCSS nested properties."

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

```shell
test.css
2:4    Expected a leading zero (number-leading-zero)
```
